package.json

```diff
@@ -0,0 +1,99 @@
+{
+    "name": "@goparrot/chowly-api-client",
+    "version": "0.0.1",
+    "description": "Typescript library which implements Chowly API",
+    "engines": {
+        "node": ">=10"
+    },
+    "main": "src/index.js",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/goparrot/chowly-api-client.git"
+    },
+    "husky": {
+        "hooks": {
+            "commit-msg": "commitlint -E HUSKY_GIT_PARAMS && npm run pre-commit",
+            "post-commit": "git update-index --again"
+        }
+    },
+    "lint-staged": {
+        "*.{ts,tsx,json,js,jsx}": [
+            "npm run format:base"
+        ]
+    },
+    "config": {
+        "commitizen": {
+            "path": "cz-conventional-changelog"
+        }
+    },
+    "scripts": {
+        "commit": "git-cz",
+        "test": "mocha --timeout 30000 'test/**/*spec.ts'",
+        "test:all": "mocha --timeout 15000 'test/**/*.spec.ts'",
+        "test:unit": "mocha 'test/unit/**/*.spec.ts'",
+        "test:e2e": "mocha 'test/e2e/**/*.spec.ts'",
+        "test:integration": "mocha --timeout 15000 'test/integration/**/*.spec.ts'",
+        "coverage": "nyc npm run test",
+        "coverage:all": "nyc npm run test:all",
+        "build:dist": "rm -rf dist/* && tsc --project tsconfig.build.json",
+        "remark": "remark README.md CHANGELOG.md CONTRIBUTING.md CODE_OF_CONDUCT.md .github/ -o -f -q && git add .",
+        "lint": "npm run lint:base -- '.'",
+        "lint:base": "npm run lint:config:check && eslint --ignore-path .gitignore --ext .ts,.tsx,.json,.js,.jsx",
+        "lint:config:check": "eslint --print-config src/index.ts | eslint-config-prettier-check",
+        "format": "npm run format:base -- '.'",
+        "format:base": "npm run lint:base -- --fix",
+        "format:staged": "git add . && lint-staged",
+        "pre-commit": "git add . && npm run format:staged && npm run lint && npm run coverage && npm run build:dist",
+        "prepublishOnly": "echo \"use 'npm run publish'\" && exit 1",
+        "publish": "npm run build:dist && node bin/prepublish.js && npm publish dist",
+        "publish:dev": "npm run publish --tag dev",
+        "publish:dev:dry": "npm run publish:dev --dry-run",
+        "version": "echo \"use 'npm run release'\" && exit 1",
+        "release": "standard-version && git push && git push --tags && npm run publish && npm run github-release",
+        "release:dry": "npm run publish:dev:dry && standard-version --dry-run"
+    },
+    "author": "Panevschi Ruslan",
+    "license": "MIT",
+    "devDependencies": {
```

Comment:
add `axios` and `class-transformer` to `peerDependencies` specifying the minimum acceptable versions